src/abaci.sol

```diff
@@ -149,6 +163,14 @@ contract StairstepExponentialDecrease is Abacus {
         }
     }
 
+    // top: initial price
+    // dur: seconds since the auction has started
+    // step: seconds between a price drop
+    // cut: percentage to decrease per step (more accurately, the % / 100..e.g., if we want a 2.4% decrease, then cut is 0.024)
+
+    // returns: top * (cut ^ (dur / step))
+
+  
```

Comment:
@WilfredTA there are formatting inconsistencies between these two comment blocks: initial description, newlines, capitalization and using `y` as return value.